app/main/lib/shared_models/video_model.py

```diff
@@ -14,61 +15,96 @@
 import tenacity
 import tmkpy
 from sqlalchemy.orm.exc import NoResultFound
+import boto3
+from botocore.client import Config
 
 from app.main.lib.shared_models.audio_model import AudioModel
 from app.main.lib.shared_models.shared_model import SharedModel
 from app.main.lib.similarity_helpers import get_context_query, drop_context_from_record
 from app.main.lib.helpers import context_matches
+from app.main.lib import media_crud
 from app.main.lib.error_log import ErrorLog
 from app.main.lib import media_crud
 from app.main import db
 from app.main.model.video import Video
-FFMPEG_DIR = "/usr/local/bin/ffmpeg"
+from app.main.lib.presto import Presto
+
+MINIO_HOST = "minio:9000"
+def download_file_from_s3(bucket: str, filename: str, local_path: str):
+    """
+    Generic download helper for s3. Could be moved over to helpers folder...
```

Comment:
I agree, this would be better as utility function as not directly a concern of video model

Reply:
lets do it!